### PR TITLE
fix reference/cli/updating-volumes

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px I've removed `pxctl volume alerts`. It might be deprecated.

### CODE REVIEW

File: content/reference/CLI/updating-volumes.md [hunk 1]
```diff
@@ -303,10 +384,3 @@ Volume	:  970758537931791410
 		Set  0
 			Node 	 :  10.99.117.133
 ```
-
-Here is the output of the volume alerts.
-
```
Comment (harsh-px) [inline content/reference/CLI/updating-volumes.md]: Why are these alerts removed?

Comment (andreipope) [reply content/reference/CLI/updating-volumes.md]: @harsh-px 

Looks like this command was removed:

```
pxctl volume alerts
Error: Subcommand is missing or invalid. This command requires a valid sub-command.
```

```
pxctl volume --help
Manage volumes

Usage:
  pxctl volume [flags]
  pxctl volume [command]

Aliases:
  volume, v

Examples:
pxctl volume create -s 100 myVolName

Available Commands:
  access               Manage volume access by users or groups
  clone                Create a clone volume
  create               Create a volume
  delete               Delete a volume
  ha-update            Update volume HA level
  import               Import data into a volume
  inspect              Inspect a volume
  list                 List volumes in the cluster
  locate               Locate volume
  requests             Show all pending requests
  restore              Restore volume from snapshot
  snap-interval-update Update volume configuration
  snapshot             Manage volume snapshots
  stats                Volume Statistics
  update               Update volume settings
  usage                Show volume usage information
```

Comment (harsh-px) [reply content/reference/CLI/updating-volumes.md]: As discussed offline, bring this section back.

Comment (andreipope) [reply content/reference/CLI/updating-volumes.md]: @harsh-px done

**Pull request merged**